dsl/camel-jbang/camel-jbang-core/src/main/resources/templates/main-kubernetes-pom.tmpl

```diff
@@ -94,39 +94,10 @@
                     </execution>
                 </executions>
             </plugin>
-            <!-- package as runner jar -->
```

Comment:
Why is this removed? the camel-main as runtime should be a standard runtime that camel jbang can run and export. I just want to ensure that this still works and that its isolated from SB or Quarkus and that its 100% standalone.